bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx92-en.html

```diff
@@ -0,0 +1,84 @@
+{% block page_og_desc %}What’s new with Firefox{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_92_en') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  {% set landing_source = '?source=' + campaign if campaign else '' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-lg">
+      <div class="mzp-c-logo mzp-t-logo-lg mzp-t-product-vpn"></div>
+      <h1 class="wnp-main-title">Change your device’s location for <strong>next-level privacy</strong></h1>
+      <p class="wnp-main-tagline"><strong>Mozilla VPN</strong> (Virtual Private Network) protects your entire internet connection on your computer, your tablet, and even your phone. For even more security, it comes with a 30-day money back guarantee.</p>
+
+      <div class="wnp-main-cta">
+        <a href="{{ url('products.vpn.landing') }}{{ landing_source }}" class="mzp-c-button mzp-t-product mzp-t-lg" rel="external" data-cta-text="Get Mozilla VPN now" data-cta-type="button">Get Mozilla VPN now</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="mzp-l-columns mzp-t-columns-three">
+        {% call picto(
+            title='Stay private on public wifi',
+            body=True,
+            image_url='img/firefox/whatsnew/whatsnew92/vpn-private.svg',
+            image_width='330'
+        ) %}
+          <p>Go ahead — check your bank account at the coffee shop. Mozilla VPN provides a secure internet connection for your entire device everywhere you go.</p>
+        {% endcall %}
+
+        {% call picto(
+            title='It’s lightning fast',
+            body=True,
+            image_url='img/firefox/whatsnew/whatsnew92/vpn-speed.svg',
+            image_width='330'
+        ) %}
+          <p>Our WireGuard® powered servers are built for speed so you can stream, download, and game as usual — knowing you’re protected.</p>
+        {% endcall %}
+
+        {% call picto(
+            title='One tap to privacy',
+            body=True,
+            image_url='img/firefox/whatsnew/whatsnew92/vpn-one-tap.svg',
+            image_width='330'
+        ) %}
+          <p>Activate Mozilla VPN with just one tap or click and get an instant, secure connection to one of our 400+ servers located around the world.</p>
+        {% endcall %}
+      </div>
+    </div>
+  </section>
+
+  <aside class="mzp-l-content c-utilities">
+    {% set releasenotes_url = url('firefox.releases.index') if not version else '/firefox/%s/releasenotes/'|format(version) %}
+    <p>{{ ftl('whatsnew-release-notes', url=releasenotes_url) }}</p>
+  </aside>
+
+</main>
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('firefox_whatsnew_92_en') }}
```

Comment:
I think you can use an exsiting bundle for this page as all that's needed is the uptodate.js script.
```suggestion
  {{ js_bundle('firefox_whatsnew_update') }}
```